src/models/user/InitialInformationRequestBody.ts

```diff
@@ -1,5 +1,5 @@
 import { NO_USER_UID_ERROR_MESSAGE } from "@/constants/message";
-
+//TODO: 클래스명 바꾸기
 export class InitialInformationRequestBody {
```

Comment:
클래스이름이 초기정보만 얻는게 아니기 때문에 수정되어야할 것 같아여
```suggestion
export class UserGetRequestBody {
```